sdk/resourcemover/Azure.ResourceManager.ResourceMover/tests/ResourceMoverManagementTestBase.cs

```diff
@@ -30,12 +34,55 @@ public void CreateCommonClient()
             Client = GetArmClient();
         }
 
-        protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionResource subscription, string rgNamePrefix, AzureLocation location)
+        protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionResource subscription, string rgName, AzureLocation location)
         {
-            string rgName = Recording.GenerateAssetName(rgNamePrefix);
             ResourceGroupData input = new ResourceGroupData(location);
             var lro = await subscription.GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Completed, rgName, input);
             return lro.Value;
         }
+
+        protected async Task<MoverResourceSetResource> CreateMoverResourceSet(ResourceGroupResource rg, string moverResourceSetName)
+        {
+            MoverResourceSetData input = new MoverResourceSetData(AzureLocation.EastUS2)
+            {
+                Properties = new MoverResourceSetProperties(AzureLocation.EastUS, AzureLocation.EastUS2),
+                Identity = new ManagedServiceIdentity(ManagedServiceIdentityType.SystemAssigned)
+            };
+            var lro = await rg.GetMoverResourceSets().CreateOrUpdateAsync(WaitUntil.Completed, moverResourceSetName, input);
+            return lro.Value;
+        }
+
+        protected async Task<VirtualNetworkResource> CreareVirtualNetwork(ResourceGroupResource rg, string vnetName)
+        {
+            VirtualNetworkData virtualNetworkData = new VirtualNetworkData()
+            {
+                Location = AzureLocation.EastUS,
+                Subnets =
+                {
+                    new SubnetData()
+                    {
+                        Name = "Subnet-0824",
+                        AddressPrefix = "10.0.0.0/24"
+                    }
+                }
+            };
+            virtualNetworkData.AddressPrefixes.Add("10.0.0.0/16");
+            var lro = await rg.GetVirtualNetworks().CreateOrUpdateAsync(WaitUntil.Completed, vnetName, virtualNetworkData);
+            return lro.Value;
+        }
+
+        protected async Task<MoverResource> CreateMoverResource(MoverResourceSetResource moverResourceSet, ResourceIdentifier vnetId, string moverResourceName)
+        {
+            string targetVnetName = Recording.GenerateAssetName("targetVnet-");
+            MoverResourceData input = new MoverResourceData
+            {
+                Properties = new MoverResourceProperties(vnetId)
+                {
+                    ResourceSettings = new VirtualNetworkResourceSettings(targetVnetName)
```

Comment:
Can we BTW rename this `VirtualNetworkResourceSettings` class? Maybe prepend a `mover` prefix?

Reply:
Renamed